pkgs/development/libraries/urdfdom/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkgconfig, tinyxml, boost
+, urdfdom-headers, console-bridge }:
+
+stdenv.mkDerivation rec {
+  pname = "urdfdom";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "ros";
+    repo = pname;
+    rev = version;
+    sha256 = "0wambq06d7dvja25zcv4agc055q9rmf3xkrnxy8lsf4nic7ra2rr";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ tinyxml boost ];
+  propagatedBuildInputs = [ urdfdom-headers console-bridge ];
+
+  cmakeFlags = [ "-DCMAKE_INSTALL_LIBDIR:PATH=lib" ];
```

Comment:
Just to make sure I guessed correctly (it might worth putting this in a comment): Upstream assumes this cmake variable is pointing to a relative path to prefix and not the absolute path where Nix' is putting the absolute path there by default?